packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -389,6 +389,8 @@ class ScrollableBaseNative extends Component {
 			if (!this.deferScrollTo) {
 				this.scrollTo(this.scrollToInfo);
 			}
+		} else if (rtl !== prevState.rtl) {
```

Comment:
I think we can handle the position in VirtualListBase only.

Reply:
I re-implemented not to change Scrollables.